rest-service/manager_rest/server.py

```diff
@@ -155,26 +155,9 @@ def reset_state(configuration=None):
     with open(os.environ['MANAGER_REST_CONFIG_PATH']) as f:
         yaml_conf = yaml.load(f.read())
     obj_conf = config.instance()
-    if 'db_address' in yaml_conf:
-        obj_conf.db_address = yaml_conf['db_address']
-    if 'db_port' in yaml_conf:
-        obj_conf.db_port = yaml_conf['db_port']
-    if 'file_server_root' in yaml_conf:
-        obj_conf.file_server_root = yaml_conf['file_server_root']
-    if 'file_server_base_uri' in yaml_conf:
-        obj_conf.file_server_base_uri = yaml_conf['file_server_base_uri']
-    if 'file_server_blueprints_folder' in yaml_conf:
-        obj_conf.file_server_blueprints_folder = \
-            yaml_conf['file_server_blueprints_folder']
-    if 'file_server_uploaded_blueprints_folder' in yaml_conf:
-        obj_conf.file_server_uploaded_blueprints_folder = \
-            yaml_conf['file_server_uploaded_blueprints_folder']
-    if 'file_server_resources_uri' in yaml_conf:
-        obj_conf.file_server_resources_uri = \
-            yaml_conf['file_server_resources_uri']
-    if 'rest_service_log_path' in yaml_conf:
-        obj_conf.rest_service_log_path = \
-            yaml_conf['rest_service_log_path']
+    for key, value in yaml_conf.iteritems():
```

Comment:
This should be yaml_conf.items() rather than iteritems(). iteritems is deprecated.
wonderful consolidation though.